src/renderers/webvr/WebXRManager.js

```diff
@@ -135,6 +135,12 @@ function WebXRManager( renderer, gl ) {
 			session.addEventListener( 'selectend', onSessionEvent );
 			session.addEventListener( 'end', onSessionEnd );
 
+			if ( gl.makeXRCompatible !== undefined ) {
```

Comment:
Actually, I might be a little bit too paranoid to check for presence of makeXRCompatible. If a Browser supports WebXR in the latest spec, then this method must be there.